File: app/src/main/java/com/sakuna63/tumbin/data/model/response/PostsResponse.java
package com.sakuna63.tumbin.data.model.response;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.sakuna63.tumbin.data.model.Blog;
import com.sakuna63.tumbin.data.model.Post;

import java.util.List;

@JsonIgnoreProperties(ignoreUnknown = true)
public class PostsResponse {
    private Blog blog;

    public Blog getBlog() {
        return blog;
    }
}

File: app/src/main/java/com/sakuna63/tumbin/data/model/response/PostsResponse.kt
package com.sakuna63.tumbin.data.model.response

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty
import com.sakuna63.tumbin.data.model.Blog
import com.sakuna63.tumbin.data.model.Post

@JsonIgnoreProperties(ignoreUnknown = true)
class PostsResponse
@JsonCreator
constructor(@JsonProperty("blog") val blog: Blog?, @JsonProperty("posts") val posts: List<Post>)

